assets/js/blocks/cart-checkout/cart/full-cart/index.js

```diff
@@ -27,72 +27,12 @@ import CartLineItemsTable from './cart-line-items-table';
 import './style.scss';
 import './editor.scss';
 
-/**
- * Given an API response with cart totals, generates an array of rows to display in the Cart block.
- *
- * @param {Object} cartTotals - Cart totals data as provided by the API.
- * @returns {Object[]} Values to display in the cart block.
- */
-const getTotalRowsConfig = ( cartTotals ) => {
-	const totalItems = parseInt( cartTotals.total_items, 10 );
-	const totalItemsTax = parseInt( cartTotals.total_items_tax, 10 );
-	const totalRowsConfig = [
-		{
-			label: __( 'List items:', 'woo-gutenberg-products-block' ),
-			value: DISPLAY_PRICES_INCLUDING_TAXES
-				? totalItems + totalItemsTax
-				: totalItems,
-		},
-	];
-	const totalFees = parseInt( cartTotals.total_fees, 10 );
-	if ( totalFees > 0 ) {
-		const totalFeesTax = parseInt( cartTotals.total_fees_tax, 10 );
-		totalRowsConfig.push( {
-			label: __( 'Fees:', 'woo-gutenberg-products-block' ),
-			value: DISPLAY_PRICES_INCLUDING_TAXES
-				? totalFees + totalFeesTax
-				: totalFees,
-		} );
-	}
-	const totalDiscount = parseInt( cartTotals.total_discount, 10 );
-	if ( totalDiscount > 0 ) {
-		const totalDiscountTax = parseInt( cartTotals.total_discount_tax, 10 );
-		totalRowsConfig.push( {
-			label: __( 'Discount:', 'woo-gutenberg-products-block' ),
-			value: DISPLAY_PRICES_INCLUDING_TAXES
-				? totalDiscount + totalDiscountTax
-				: totalDiscount,
-		} );
-	}
-	if ( ! DISPLAY_PRICES_INCLUDING_TAXES ) {
-		const totalTax = parseInt( cartTotals.total_tax, 10 );
-		totalRowsConfig.push( {
-			label: __( 'Taxes:', 'woo-gutenberg-products-block' ),
-			value: totalTax,
-		} );
-	}
-	const totalShipping = parseInt( cartTotals.total_shipping, 10 );
-	const totalShippingTax = parseInt( cartTotals.total_shipping_tax, 10 );
-	totalRowsConfig.push( {
-		label: __( 'Shipping:', 'woo-gutenberg-products-block' ),
-		value: DISPLAY_PRICES_INCLUDING_TAXES
-			? totalShipping + totalShippingTax
-			: totalShipping,
-		description: __(
-			'Shipping to location (change address)',
-			'woo-gutenberg-products-block'
-		),
-	} );
-
-	return totalRowsConfig;
-};
-
 // @todo this are placeholders
 const onActivateCoupon = ( couponCode ) => {
 	// eslint-disable-next-line no-console
 	console.log( 'coupon activated: ' + couponCode );
 };
-const cartTotals = {
+const fetchedCartTotals = {
```

Comment:
Is this inline test data needed? We have `previews/cart-items.js` which should contain a valid API response. If you need different data for testing, consider editing the preview data.

Reply:
Gotcha. Could potentially add this test data to a new `previews/cart` to match the `store/cart` endpoint. But yep I see that I'm commenting on stuff that came in a previous PR, apologies :)

Reply:
FYI @senadir I've shifted the preview cart totals data in

Reply:
I can help you rebase if this was merged before your, or I can rebase mine, no problem